### PR TITLE
Add node e2e ci for cri-containerd.

### PR DESCRIPTION
**Pull request by Random-Liu**
@krzyzacy

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -72,26 +72,17 @@ presubmits:
         - "--repo=github.com/kubernetes-incubator/$(REPO_NAME)=$(PULL_REFS)"
         - "--root=/go/src"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--clean"
```
Comment (Random-Liu) [inline prow/config.yaml]: Is the `clean` needed?

File: prow/config.yaml [hunk 2]
```diff
@@ -72,26 +72,17 @@ presubmits:
...
         env:
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/service-account/service-account.json
         volumeMounts:
         - name: service
           mountPath: /etc/service-account
           readOnly: true
-        - name: cache-ssd
```
Comment (Random-Liu) [inline prow/config.yaml]: Do we need ssd cache?

File: prow/config.yaml [hunk 3]
```diff
@@ -3185,6 +3176,76 @@ postsubmits:
...
+
+- name: ci-cri-containerd-node-e2e
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20171008-cd94f30a-master
+      args:
+      - --root=/go/src
+      - "--git-cache=/root/.cache/git"
```
Comment (Random-Liu) [inline prow/config.yaml]: I don't see `git-cache` is used in the code, do we still need the ssd cache?

File: prow/config.yaml (hunk 1 above)
Comment (krzyzacy) [reply prow/config.yaml]: hummm it's mostly for git cache, since you are not using it I think we can remove it.

File: prow/config.yaml (hunk 3 above)
Comment (krzyzacy) [reply prow/config.yaml]: it's in https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L225, but you don't have to use it

**Comment by Random-Liu** [comment]
@krzyzacy Hm, then I'll keep the code as it is given your suggestion.

**Comment by krzyzacy** [comment]
keep the pr and ci job consistent (either use git-cache or not use git-cache), otherwise lgtm

**Comment by Random-Liu** [comment]
@krzyzacy They are consistent now.

Build doesn't use cache, test does. CI and pr are the same.

**Comment by krzyzacy** [comment]
okay :-)

/lgtm

**Pull request merged**